records_mover/url/base.py

```diff
@@ -201,7 +202,9 @@ def open(self, mode: str = "rb") -> IO[Any]:
         "exist in the directory."
         raise NotImplementedError()
 
-    def wait_to_exist(self) -> None:
+    def wait_to_exist(self,
+                      log_level: int = logging.INFO,
+                      ms_between_polls: int = 50) -> None:
```

Comment:
Let's take this existing method and make it more configurable so we get the UX and AWS API friendliness we want.

Reply:
Love a UX update, very philosophical function name here.